homeassistant/components/media_player/panasonic_viera.py

```diff
@@ -26,12 +28,13 @@
 SUPPORT_VIERATV = SUPPORT_PAUSE | SUPPORT_VOLUME_STEP | \
     SUPPORT_VOLUME_SET | SUPPORT_VOLUME_MUTE | \
     SUPPORT_PREVIOUS_TRACK | SUPPORT_NEXT_TRACK | \
-    SUPPORT_TURN_OFF
+    SUPPORT_TURN_OFF | SUPPORT_TURN_ON
```

Comment:
@balloob Python syntax is new for me, would something like
```python
if CONF_MAC:
SUPPORT_VIERATV = SUPPORT_VIERATV | SUPPORT_TURN_ON
```
would be fine or such check must be done inside ```def setup_platform```` ?